wandb/cli/cli.py

```diff
@@ -2700,7 +2719,10 @@ def restore(ctx, run, no_git, branch, project, entity):
     return commit, json_config, patch_content, repo, metadata
 
 
-@cli.command("online", help="Enable W&B sync")
+@cli.command(
+    "online",
+    help="Enable live syncing with W&B when a network connection is available.",
```

Comment:
I'd be confused about the "when a network connection is available" part. I'd wonder whether I can only use `wandb online` if a network connection is available or if it's a no-op when a network connection is unavailable. But actually it's neither: it just undoes `wandb offline`. What if this docstring just said that?